packages/syft/src/syft/protocol/releases/0.8.3.json

```diff
@@ -0,0 +1,194 @@
+{
```

Comment:
We would need to add this folder in cd-syft.yml to be commit during Syft stable releases.

in the step called

`Changes to commit to Syft Repo during stable release`